commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java

```diff
@@ -40,6 +42,14 @@ public static void setup() {
 
   }
 
+  @AfterEach
+  public void signout() {
+    ManageLogInOut manageLogInOut;
+    manageLogInOut=new ManageLogInOut(this);
```

Comment:
One line is enough : ManageLogInOut manageLogInOut = new ManageLogInOut(this);

Reply:
done